src/lib/emails/templates/TicketReceiverEmail.tsx

```diff
@@ -152,4 +91,16 @@ const hr = {
 const footer = {
   color: "#8898aa",
   fontSize: "12px"
-};
\ No newline at end of file
+};
+
+const renderTicketReceiverEmail = async ({ eventName, ticketUrl, imageUrl }) => {
```

Comment:
new paradigm - at the end of each template file (lib/emails/teampltes), add the renderer function - render from `react-email` will work only on .tsx files, therefore cannot use on .ts API routes